pyccl/halos/profiles/profile_base.py

```diff
@@ -47,35 +47,31 @@ def __init__(self):
                             "_real or _fourier implementation.")
         self.precision_fftlog = FFTLogParams()
 
-    @property
-    @abstractmethod
-    def normprof(self) -> bool:
-        """Normalize the profile in auto- and cross-correlations by
-        :math:`I^0_1(k\\rightarrow 0, a|u)`
-        (see :meth:`~pyccl.halos.halo_model.HMCalculator.I_0_1`).
-        """
-
-    # TODO: CCLv3 - Rename & allocate _normprof_bool to the subclasses.
+    def get_normalization(self, cosmo, a, hmc):
+        """Profiles may be normalised by an overall function of redshift
+        (or scale factor). This function may be cosmology dependent and
+        often comes from integrating certain halo properties over mass.
+        This method returns this normalising factor. For example,
+        to get the normalised profile in real space, one would call
+        the `real` method, and then **divide** the result by the value
+        returned by this method.
```

Comment:
I would personally write here in the docstring the function you showed me in the halos PR.
Use `r` in the beginning of the docstring to avoid having to put double \\ in math mode.

Reply:
Will do this in the docs PR